### PR TITLE
[NV-PHP-6]API: Add Blueprints Feature

### PR DESCRIPTION
**Pull request by TheAbhishekIN**
I have added the blueprints feature, please review the PR and let me know if there any changes required

### CODE REVIEW

File: src/Actions/ManagesBlueprints.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+trait ManagesBlueprints
+{
+    /**
+     * Fetch list of blueprints [ Come back to this for pagination---->]
+     *
+     * @return Blueprint
+     */
+    public function getBlueprintsGroupByCategory(): Blueprint
+    {
+        $blueprints = $this->get("blueprints/group-by-category");
```
Comment (unicodeveloper) [inline src/Actions/ManagesBlueprints.php:16]: Please give a newline space before the return

File: src/Actions/ManagesBlueprints.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Fetch blueprints by templateId
+     *
+     * @param string $templateId
+     * @return Blueprint
+     */
+    public function getBlueprints(string $templateId): Blueprint
+    {
+        $blueprint = $this->get("blueprints/{$templateId}")['data'];
```
Comment (unicodeveloper) [inline src/Actions/ManagesBlueprints.php:28]: Please give a newline space before the return

File: src/Actions/ManagesBlueprints.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        $blueprints = $this->get("blueprints/group-by-category");
+        return new Blueprint($blueprints, $this);
+    }
+
+    /**
+     * Fetch blueprints by templateId
+     *
+     * @param string $templateId
+     * @return Blueprint
```
Comment (unicodeveloper) [inline src/Actions/ManagesBlueprints.php:24]: Use the full path in the @return. like this `@return Novu\SDK\Resources\Blueprint`

File: src/Actions/ManagesBlueprints.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+namespace Novu\SDK\Actions;
+
+use Novu\SDK\Resources\Blueprint;
+
+trait ManagesBlueprints
+{
+    /**
+     * Fetch list of blueprints [ Come back to this for pagination---->]
+     *
+     * @return Blueprint
```
Comment (unicodeveloper) [inline src/Actions/ManagesBlueprints.php:12]: Use the full path in the @return. like this `@return Novu\SDK\Resources\Blueprint`

**Comment by TheAbhishekIN** [comment]
Hi @unicodeveloper ,
Thank you for the feedback, 
I have fixed the Feedback, please check and let me know if there's anything for me

**Comment by unicodeveloper** [comment]
LGTM ⭐ 🔥 @TheAbhishekIN

**Review by unicodeveloper — APPROVED** [review]

**Comment by unicodeveloper** [comment]
This closes #51

**Pull request merged**